intelmq/tests/lib/test_pipeline.py

```diff
@@ -75,7 +76,7 @@ def test_send(self):
 
     def test_receive_unicode(self):
         self.pipe.state['test-bot-input'] = [SAMPLES['unicode'][0]]
-        self.assertEqual(SAMPLES['unicode'][1], self.pipe.receive())
+        self.assertEqual(SAMPLES['unicode'][1], utils.decode(self.pipe.receive()))
```

Comment:
I'm not sure if that testcase still makes sense. Is unicode data in the pipeline valid at all now?